### PR TITLE
circular torus

### PR DESCRIPTION
**Pull request by dkatelin1702**
added circular torus CSG to CAD comparison

### CODE REVIEW

File: src/model_benchmark_zoo/circulartorus.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        )
+        universe = openmc.DAGMCUniverse(filename).bounded_universe()
+        geometry = openmc.Geometry(universe)
+
+        model = openmc.Model(geometry=geometry)
+        return model
+
+    
+    
+
```
Comment (shimwell) [inline src/model_benchmark_zoo/circulartorus.py:51]:
```suggestion
```

**Comment by shimwell** [comment]
Many thanks @dkatelin1702 this looks great. We could just do with a ci test file for this new shape, perhaps take a look at these existing ones

https://github.com/fusion-energy/model_benchmark_zoo/tree/main/.github/workflows

**Comment by dkatelin1702** [comment]
added the ci test

**Comment by shimwell** [comment]
Super, thanks for adding the tests.

One last request.

Could you add an image to the README.md file like these lines of code
https://github.com/fusion-energy/model_benchmark_zoo/blob/3234e12953dab54f72c6f73ac1bb2542a9b73860/README.md?plain=1#L12

File: tests/test_csg_cad_circulartorus.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from model_benchmark_zoo import Circulartorus
+import openmc
+import math
+
```
Comment (shimwell) [inline tests/test_csg_cad_circulartorus.py:4]: CI tests are failing with NameError: name 'np' is not defined, this should fix the error

```suggestion
import numpy as np
```

**Comment by shimwell** [comment]
Super the tests are passing. Let us merge this in. I can add the readme diagram

**Pull request merged**